tools/ci_build/github/azure-pipelines/templates/jar-maven-signing-win.yml

```diff
@@ -0,0 +1,71 @@
+parameters:
+  - name: JarFileDirectory
+    type: string
+
+  - name: WorkingDirectory
+    type: string
+
+  - name: EnableJarMavenSigning
+    type: boolean
+    default: false
+
+steps:
+  - task: AzureKeyVault@2
+    displayName: 'Get GnuPG signing keys'
+    condition: and(succeeded(), eq('${{ parameters.EnableJarMavenSigning }}', true))
+    inputs:
+      azureSubscription: 'OnnxrunTimeCodeSign_20240611'
+      KeyVaultName: 'ort-release'
+      SecretsFilter: 'java-pgp-pwd,java-pgp-key'
+      RunAsPreJob: false
+
+  - task: PowerShell@2
+    displayName: 'Sign jar files: GnuPG and sha256'
+    condition: and(succeeded(), eq('${{ parameters.EnableJarMavenSigning }}', true))
+    inputs:
+      targetType: 'inline'
+      workingDirectory: '$(Build.BinariesDirectory)'
+      script: |
```

Comment:
If you move the content to a separate *.sh file, we have a pipeline that can do grammar check and static analysis on the file.

Reply:
Previously used ps1 file, but it cannot recognize the secrets from key vault, have tried bunch of formats like $(SECRET_NAME), '$(SECRET_NAME)',  and so on..... Seems only inline will recognize the fetched secret names.